packages/blade/src/components/Alert/StyledAlert.web.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+import type { StyledAlertProps } from './StyledAlert.d';
+import { makeSpace } from '~utils';
+
+const StyledAlert = styled.div<StyledAlertProps>((props) => {
+  const { theme, contrastType, intent, isFullWidth, isBorderless } = props;
+  const _isFullWidth = isFullWidth || isBorderless; // a borderless alert is automatically set to full width
+  const feedbackColors = theme.colors.feedback;
+
+  return {
+    background: feedbackColors.background[intent][contrastType],
+    padding: `${makeSpace(theme.spacing[3])} ${makeSpace(theme.spacing[4])}`,
+    borderRadius: isBorderless ? theme.border.radius.none : theme.border.radius.medium,
+    borderColor: feedbackColors.border[intent][contrastType],
+    borderWidth: isBorderless ? theme.border.width.none : theme.border.width.thin,
+    borderStyle: 'solid',
+    display: 'flex',
+    flexDirection: 'row',
+    maxWidth: _isFullWidth ? 'auto' : 584,
```

Comment:
How will 584 translate to mobile devices? Or will the alert always be full width in mobile? (which tbh makes sense)

Reply:
Same as on web, 584 effective px